src/browser/cef_browser.py

```
@@ -222,13 +239,27 @@ def minimize(self) -> None:
+                    end tell
+                    '''
+                    os.system(f"osascript -e '{script}'")
                 else:  # Linux
-                    # On Linux, we can use xdotool to minimize the window
-                    # This requires xdotool to be installed
-                    os.system(f"xdotool search --name \"{self.title}\" windowminimize")
+                    # Try multiple approaches for Linux
+                    # First try by window title
+                    os.system(f"xdotool search --name \"{self.title}\" windowminimize 2>/dev/null || true")
+                    # Then try by class name if title fails
+                    os.system(f"xdotool search --class \"Chromium\" windowminimize 2>/dev/null || true")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)